From 693ca2d71217ee1e33e7a9dbdc24bd8b9f1c6a7d Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Sat, 6 Oct 2018 20:22:26 +0800
Subject: [PATCH] arm64: dts: rockchip: enable usb on rk1808 evb

This patch enables usb OHCI/EHCI/xHCI controllers
and usb 2.0 phy.

Change-Id: I77671a75a1291b4fcf949a429d3c8cc58ce5b1a3
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi | 28 ++++++++++++++++++++
 1 file changed, 28 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index 922428b56025..e4b7b80a3f1a 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -587,6 +587,34 @@
 	status = "okay";
 };
 
+&u2phy {
+	status = "okay";
+};
+
+&u2phy_host {
+	status = "okay";
+};
+
+&u2phy_otg {
+	status = "okay";
+};
+
+&usb_host0_ehci {
+	status = "okay";
+};
+
+&usb_host0_ohci {
+	status = "okay";
+};
+
+&usbdrd3 {
+	status = "okay";
+};
+
+&usbdrd_dwc3 {
+	status = "okay";
+};
+
 &pinctrl {
 	pmic {
 		pmic_int: pmic_int {
-- 
2.35.3

